gradle/wrapper/gradle-wrapper.properties

```diff
@@ -4,4 +4,3 @@ distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
 distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.3-all.zip
-x
```

Comment:
Woot?

Reply:
that's a harmless typo. Cleanup is coming.